DbService/inc/GrlTool.hh

```diff
@@ -0,0 +1,69 @@
+#ifndef DbService_GrlTool_hh
+#define DbService_GrlTool_hh
+
+// This class provides an interface to the goodrun database
+// There are two main interfaces: bits and list
+
+#include "Offline/DbService/inc/DbReader.hh"
+#include "Offline/DbService/inc/DbSql.hh"
+#include "Offline/DbService/inc/GrlBit.hh"
+#include "Offline/DbService/inc/GrlEntry.hh"
+#include "Offline/DbService/inc/GrlList.hh"
+#include "Offline/DbService/inc/GrlWord.hh"
+#include "Offline/DbTables/inc/DbId.hh"
+#include "Offline/GeneralUtilities/inc/StringVec.hh"
+#include <string>
+
+namespace mu2e {
+
+class GrlTool {
+ public:
+  GrlTool();
+
+  // define a quality word
+  int defineWord(const std::string& name, const std::string& desc = "");
+  // define bits within the word
+  int defineBit(const std::string& name, const std::string& bitname,
+                const std::string& desc = "");
+  // set quality bits for a word, if bits is empty, then no bits set, all good
+  int setBits(const std::string& name, const DbIoV& iov,
```

Comment:
Do you intend this interface allowing creation/definition of grl bits to be open to all? If not, how do you forsee administering access?

Reply:
Database write access is determined by the user, who has to have a kerberos identity and a database account with the right privilege.  Currently, only mu2epro can write, but I imagine an online shift or ops account would be another obvious candidate.  Maybe certain people from analysis groups.  As it is now, anyone with write access can write to any GRL, but if there is a need to narrow privileges, we can do that. Most errors could be repaired by an admin too.